db/migrate/20180620170052_add_region_to_file_depot.rb

```diff
@@ -0,0 +1,5 @@
+class AddRegionToFileDepot < ActiveRecord::Migration[5.0]
+  def change
+    add_column :file_depots, :region, :string
```

Comment:
A region column feels specific to amazon and not to depots.  It's also a string... like the mistake we made with using zone name's in the queue instead of id's referencing rows in the zone table.  If we intend to use depots in other clouds, it might make sense to consider using a polymorphic relationship to that provider's storage location table.

This might be out of scope of this PR but I see the zone mistake we made with the queue when I look at this.

Reply:
Yes, it is really just for the S3 File Depot, but there is not a separate S3 File Depot table.  There will also be a depot for Swift.  Can we look at your suggestion at a later point so that we can get this merged for the upcoming Sprint?  There are 4 other PRs that are waiting like dominoes on this one to fall... (poor choice of words, I know)

Reply:
Sure, unless we know how it will look for other systems, it's probably better to wait until we know how it will look.  That is, unless the migration would be painful.

Reply:
No we don't know at this point.  Thanks for your help.